db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+{
+  admin: "Account with administrative rights",
+  moderator: "Moderator role. Allows to update other's posts and comments",
+  editor: "Account is allowed to create own posts, publish them and update them"
```

Comment:
I think it should be in localisation. And getting from there like `I18n.t("roles.descriptions.#{role_type}")`